### ISSUE TITLE
fix: Report errors from call to model

### ISSUE THREAD

**Issue by powersj** [rafaeldtinoco/ghtimecardator#4]
Hey @rafaeldtinoco,

I wanted to try this out, but kept getting nothing back from:

```s
go run . last-3days detailed influxdata/telegraf
```

When clearly I could see it paging through updates in GitHub based on the limited output. I noticed you were ignoring any error from the call to [OpenAI here](https://github.com/rafaeldtinoco/ghtimecardator/blob/main/ghtimecardator.go#L413). Once I printed the error I noticed:

```s
-API returned unexpected status code: 404: The model `gpt-4` does not exist or you do not have access to it. Learn more: 
```

Might be worth reporting errors that you get back or even stop execution since sending more requests will probably not clean up any additional failures.

**Comment by rafaeldtinoco** [comment]
Yep. Might be usefull allowing to change the model as well! Thx for trying ;).
